src/settings_type.h

```diff
@@ -318,6 +318,7 @@ struct NetworkSettings {
 	uint8_t       max_companies;                            ///< maximum amount of companies
 	uint8_t       max_clients;                              ///< maximum amount of clients
 	TimerGameCalendar::Year restart_game_year;            ///< year the server restarts
+	uint16_t      restart_hours;                            ///< number of hours to run the server before automatic restart
```

Comment:
Why not `std::chrono::hours` as type?

Reply:
Yes ofc you silly goose. There is a bit more work that needs to be done for this :D But only stating what line in fmt fails is not useful; you need to check what line causes that :)

What I am guessing, is that you see the error from `Debug()`. You have to tell him how to print time. There are several ways, the easiest is to use `count()`.

Either way, our settings-code can't handle `std::chrono`, so let this PR not be the first to introduce that. Bit sad, but it is what it is.